src/containers/Cfe/index.tsx

```diff
@@ -128,103 +130,146 @@ class Cfe extends React.Component<CfeProps, CfeState> {
             styles.content,
             { [styles.hidden]: currentTab !== VIEWER_TAB_KEY },
         ]);
-        const plotClassNames = classNames([{ [styles.hidden]: currentTab === VIEWER_TAB_KEY }]);
+        const plotClassNames = classNames([
+            styles.content,
+            { [styles.hidden]: currentTab === VIEWER_TAB_KEY },
+        ]);
 
-        return (
+        type MenuItem = Required<MenuProps>["items"][number];
+        const menuItems: MenuItem[] = [
+            {
+                label: (
+                    <>
+                        <span className={classNames(["icon-moon", "anticon", styles.plotIcon])} />
+                        Plot
+                    </>
+                ),
+                key: PLOT_TAB_KEY,
+            },
+            {
+                label: (
+                    <>
+                        <span className={classNames(["icon-moon", "anticon", styles.cubeIcon])} />
+                        3D Viewer
+                    </>
+                ),
+                key: VIEWER_TAB_KEY,
+            },
+        ];
```

Comment:
Ant now requires menu items to be a prop instead of part of the React component tree